grader_support/gradelib.py

```diff
@@ -546,6 +546,17 @@ def __init__(self, fn_name, args, environment=None, output_writer=None, short_de
             short_desc = "Test: %s(%s)" % (fn_name, ", ".join(repr(a) for a in args))
         Test.__init__(self, test_fn, short_desc, detailed_desc, compare)
 
+class ExecWrappedStudentCodeTest(Test):
+    """
+    A Test that exec student code and capture the stdout result.
```

Comment:
Nit: `Test that executes the student code and captures the result, which is output to stdout.`